Source/Aptabase/Private/AptabaseAnalyticsProvider.cpp

```diff
@@ -79,7 +76,7 @@ void FAptabaseAnalyticsProvider::RecordEventInternal(const FString& EventName, c
 {
 	if (!bHasActiveSession)
 	{
-		UE_LOG(LogAptabase, Log, TEXT("No session is currently active. Discarding event."));
+		UE_LOG(LogAptabase, Warning, TEXT("No session is currently active. Discarding event."));
```

Comment:
Consider this could cause unexpected behaviour for the developer (he wanted to track an event but he won't be able to see it in the dashboard), I would raise it to `Warning` to give it more visibility.